experiment/config-forker/README.md

```diff
@@ -0,0 +1,64 @@
+# config-forker
+
+config-forker forks presubmit, periodic, and postsubmit job configs with the `fork-per-release` annotation.
+
+## Usage
+
+* `--job-config`: Path to the job config
```

Comment:
Does it need `--config-path` and `--job-config`?

Reply:
No, it only uses the job config.